src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import task.Task;
+
+/**
+ * Class responsible for adding and deletion of tasks
+ */
+public class TaskList {
+    private List<Task> taskList;
```

Comment:
Should this member be named in a plural form (e.g. "tasks") since it represents a collection of `Task`s?

Reply:
I will rename the field accordingly, thanks for identifying!